website/src/controllers/api-v2/groups.js

```diff
@@ -436,7 +442,8 @@ api.seenMessage = function(req,res,next){
 api.likeChatMessage = function(req, res, next) {
   var user = res.locals.user;
   var group = res.locals.group;
-  var message = _.find(group.chat, {id: req.params.mid});
+  var chat = res.locals.unchangedGroupChat;
+  var message = _.find(chat, {id: req.params.mid});
```

Comment:
Is there a way to reach into an array on a mongo doc and just update something in it? I feel like that's how this should be handled instead of how I've implemented it.

 I know how to do it if it's an object, not sure how to do it with an array.